types/universal.d.ts

```diff
@@ -87,11 +87,6 @@ export interface SpringProps<DS extends object = {}> {
    * Takes a function that receives interpolated styles
    */
   children?: SpringRendererFunc<DS>
-  /**
-   * Prevents animation if true, you can also pass individual keys
-   * @default false
-   */
-  immediate?: boolean | string[] | ((key: string) => boolean)
```

Comment:
Did `immediate` ever actually accept `string[]`? I can't find it in the code (in fact if it's an array it's just treated as always `true` in the `Controller`). This made it incompatible with `SpringBaseProps` so I had to remove the `string[]`.

Reply:
@Kovensky  No, immediate can be a boolean or a function returning boolean. This could be a left over.